src/common/utils.js

```diff
@@ -0,0 +1,5 @@
+export default class Utils {
```

Comment:
class Utils is not need, so please just export objects.
export default {
...
}

Thanks.
https://developer.mozilla.org/zh-TW/docs/Web/JavaScript/Reference/Statements/export